src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java

```diff
@@ -391,7 +390,7 @@ private void generate0()
             // TODO verify
             unlockedSetStat(LOSS_RATE_DOWNLOAD, lossRateDownload);
             // TODO verify
-            unlockedSetStat(LOSS_RATE_UPLOAD, lossRateUpload);
+            unlockedSetStat(LOSS_RATE_UPLOAD, lossRateReported);
```

Comment:
I wonder if we should call this out specifically as loss reported by TCC

Reply:
I'm in favor of it in theory, but I'm also trying to avoid having to change the existing stats names here.  (Both to avoid changing the xmpp extensions, and to avoid changing our existing DataDog reporting.)